lib/worker.js

```diff
@@ -308,6 +308,7 @@ class AssetComputeWorker {
 
         try {
             if (await this.shouldPostProcess(rendition)) {
+                this.metrics.add({ imagePostProcess: true });
```

Comment:
I thought we would also add the post-processing duration here?

Reply:
We already have a post processing duration in the metrics, which includes `shouldPostProcess` (e.g. read intermediate metadata), any extra stuff like downloading the watermark and the final imagemagick convert command.

I think that's enough for the metrics, more fine grained measurements here would not be regularly looked at anyway. We only wanted more details in the logs for individual issue analysis (where it came up).

This flag was added so we can query what amount of e.g. worker x runs are with or without post processing.